infrastructure/cdn-in-a-box/traffic_ops_data/profiles/010-ATS_EDGE_TIER_CACHE.json

```diff
@@ -87,7 +87,7 @@
 			"configFile": "records.config",
 			"name": "CONFIG proxy.config.log.logfile_dir",
 			"secure": false,
-			"value": "STRING /var/log/trafficserver"
+			"value": "STRING /opt/trafficserver/var/log/trafficserver"
```

Comment:
Shouldn't this be determined from the RPM the same way the configuration files are for the `chown` step?

Reply:
Getting logfile_dir from the RPM in 4486105741